src/cli/cargo.rs

```diff
@@ -11,13 +11,18 @@ pub fn call(args: Vec<&str>) -> Option<&'static str> {
         command.arg(arg);
     }
 
-    let output = command.stdout(Stdio::inherit())
-                        .stderr(Stdio::inherit())
-                        .output()
-                        .ok();
+    let output_result = command.stdout(Stdio::inherit())
```

Comment:
I am considering using an `if let` statement in this case, since we are only using the contents of `Ok` but not `Err`. Tell me what you think of this:

```rust
let exec_result = command.stdout(Stdio::inherit())
                         .stderr(Stdio::inherit())
                         .output();

if let Ok(output) = exec_result {
    if output.status.success() {
        None
    } else {
        Some("Cargo exited with an error!")
    }
} else {
    Some("Could not launch Cargo!")
}
```